implementations/rust/ockam/ockam_vault/src/types/secrets.rs

```diff
@@ -0,0 +1,58 @@
+use ockam_core::compat::vec::Vec;
+
+/// A handle to a secret inside a vault.
```

Comment:
I could be a good idea to give examples for what kind of bytes will end up being a secret handle

Reply:
That is implementation-specific. It's already 2 different things for AWS and Software implementations

Reply:
It's ok that you explain that to me here, but I think that would be better in a comment :-). In particular because a reader might wonder why this is not a simple `String`

Reply:
I'll add that comment in the next PR.
P.S. Phrase `simple String` triggered me :D, `Vec<u8>` is what simpler than `String`)